src/AggregateRoots/AggregateRoot.php

```diff
@@ -43,6 +43,13 @@ public static function retrieve(string $uuid)
         return $aggregateRoot->reconstituteFromEvents();
     }
 
+    public function loadUuid(string $uuid): self
+    {
+        $this->uuid = $uuid;
```

Comment:
Maybe there should be a guard here to protect an already loaded Aggregate to be re-loaded? 🤔